python/ray/autoscaler/_private/docker.py

```diff
@@ -1,73 +1,102 @@
 
-    return config
+    return None
 
 
-def with_docker_exec(cmds,
-                     container_name,
-                     docker_cmd,
-                     env_vars=None,
-                     with_interactive=False):
-    assert docker_cmd, "Must provide docker command"
-    env_str = ""
-    if env_vars:
-        env_str = " ".join(
-            ["-e {env}=${env}".format(env=env) for env in env_vars])
+def with_docker_exec(cmds: List[str],
+                     container_name: str,
+                     docker_cmd: str,
+                     with_interactive: bool = False):
+    """
+    Wraps a command so that it will be executed inside of a container.
+    """
     return [
-        "docker exec {interactive} {env} {container} /bin/bash -c {cmd} ".
-        format(
+        "docker exec {interactive} {container} /bin/bash -c {cmd} ".format(
             interactive="-it" if with_interactive else "",
-            env=env_str,
             container=container_name,
             cmd=quote(cmd)) for cmd in cmds
     ]
 
 
-def _check_helper(cname, template, docker_cmd):
+def _check_helper(container_name: str, template: str, docker_cmd: str) -> str:
+    """
+    Common functionality for running commmands to inspect some parameter.
+    The produced command string will always exit with 0.
+    Args:
+        container_name (str): Name of container to check
+        template (str): Golang style template (without brackets)
+        docker_cmd (str): Name of Docker program to use (docker|podman)
+    """
     return " ".join([
-        docker_cmd, "inspect", "-f", "'{{" + template + "}}'", cname, "||",
-        "true"
+        docker_cmd, "inspect", "-f", "'{{" + template + "}}'", container_name,
+        "||", "true"
     ])
 
 
-def check_docker_running_cmd(cname, docker_cmd):
-    return _check_helper(cname, ".State.Running", docker_cmd)
+def check_docker_running_cmd(container_name: str, docker_cmd: str) -> str:
+    return _check_helper(container_name, ".State.Running", docker_cmd)
 
 
-def check_bind_mounts_cmd(cname, docker_cmd):
-    return _check_helper(cname, "json .Mounts", docker_cmd)
+def check_bind_mounts_cmd(container_name: str, docker_cmd: str) -> str:
+    return _check_helper(container_name, "json .Mounts", docker_cmd)
 
 
-def check_docker_image(cname, docker_cmd):
-    return _check_helper(cname, ".Config.Image", docker_cmd)
+def check_docker_image(container_name: str, docker_cmd: str) -> str:
+    return _check_helper(container_name, ".Config.Image", docker_cmd)
 
 
-def docker_start_cmds(user, image, mount_dict, container_name, user_options,
-                      cluster_name, home_directory, docker_cmd):
+def docker_start_cmds(image: str, mount_dict: Dict[str, str],
+                      container_name: str, user_options: List[str],
+                      cluster_name: str, home_directory: str,
+                      docker_cmd: str) -> str:
+    """
+    Build command to start a docker container.
+    """
```

Comment:
nit: btw, the first string should always be on the first line (and should never linebreak). same applies to rest of the docstrings.
```suggestion
    """Build command to start a docker container."""
```

Reply:
Ahh did not know this!! Thanks :)